test/rpc_test.py

```diff
@@ -1109,11 +1096,6 @@ def test_rpc_join_and_shutdown(self):
         rpc.join_rpc()
         rpc.shutdown()
 
-    @dist_init
-    def test_get_default_rpc_timeout(self):
-        timeout = rpc.get_rpc_timeout()
-        self.assertEqual(timeout, rpc.constants.DEFAULT_RPC_TIMEOUT)
-
```

Comment:
Why did we remove test_get_default_rpc_timeout?

Reply:
This must've show up due to merging/rebasing issues with https://github.com/pytorch/pytorch/pull/30020/files/ - this test was removed there, and we added `test_get_rpc_timeout` instead.